pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -5074,6 +5266,36 @@ status:
 
 }
 
+func TestReconcileWithPipelineResults(t *testing.T) {
+	testCases := []struct {
+		name        string
+		embeddedVal string
+	}{
+		{
+			name:        "default embedded status",
+			embeddedVal: config.DefaultEmbeddedStatus,
+		},
+		{
+			name:        "full embedded status",
+			embeddedVal: config.FullEmbeddedStatus,
+		},
+		{
+			name:        "both embedded status",
+			embeddedVal: config.BothEmbeddedStatus,
+		},
+		{
+			name:        "minimal embedded status",
+			embeddedVal: config.MinimalEmbeddedStatus,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			runTestReconcileWithPipelineResults(t, tc.embeddedVal)
+		})
+	}
+}
+
```

Comment:
it seems that this change is unrelated 🤔

Reply:
See above.